Adding variables for the number of charging stations

jeremydumoulin opened the pull request:
New variables related to the number of charging stations for electric vehicles, defined in the context of the OpenMod4Africa project, task 3.1.

The number of variables requested is deliberately small for the time being, as it is not yet clear if a higher granularity will be required (e.g.: by also adding chargers at home, at work, or by segmenting the type of charger regarding their technical characteristics).

Many thanks for considering this request. Happy for any comments/suggestions.

sandrinecharousset commented:
@jeremydumoulin I would advise you to request some reviewers to review or this PR may wait a long time.  You could eg request @erikfilias @phackstock and myself  (@phackstock is leading this nomenclature work and is the one who can merge, @erikfilias and myself have experienced the process in openentrance )
Best

danielhuppmann commented on definitions/variable/technology/technologies.yaml:
```diff
@@ -205,6 +205,13 @@
+- Stock|Transportation|Charging Station:
+    description: Number of charging stations for electric vehicles
+    unit: million
+- Stock|Transportation|Charging Station|Public:
```
In my opinion, this variable name is not sufficiently clear whether they are in public areas (how defined?) or owned by public entities (in contrast to those offered by private companies or on private/residential areas).

Also, please create a complete sub-categorization so that public/private/other charging stations  can be reported such that they be summed up to total number of charging stations.

jeremydumoulin commented:
@sandrinecharousset : thank you for the advice, but it seems to me that I do not have enough rights to request specific reviewers. I should probably ask @phackstock for that.

@danielhuppmann : Many thanks for following up on this PR. I have added private and semi-public stations to complete the sub-categorization, and also updated the variable descriptions for clarity.
Note that the public/semi-public/private classification is a common way of differentiating charging stations according to their accessibility (see for example: [https://link.springer.com/chapter/10.1007/978-3-319-13194-8_4](https://link.springer.com/chapter/10.1007/978-3-319-13194-8_4)), although I agree that in some cases a higher level of detail may indeed be useful.

Best regards

danielhuppmann reviewed (commented):
The distinction between public, semi-public and private makes sense to me.

One final thought - does it make sense to name the variable as "Infrastructure|Transportation|Charging Station|...". It seems to be a bit different in nature from a "stock" like vehicles...

jeremydumoulin commented:
Thanks for the quick feedback.

Good point. I probably do not have enough experience with variable names to know which is the best option here. It seems that having "Infrastructure|Transportation|Charging Station|..." is indeed more relevant. 
It would also be possible, I suppose, to have something very general like "Number|Charging station|..." as for the number of heat pumps ("Number|{Residential/Commercial}|Heat Pumps"), but I prefer your suggestion. I have updated the PR.

Best

danielhuppmann reviewed (approved):
Looks good to me, thanks! Will merge if @sandrinecharousset also approves.

phackstock commented:
@jeremydumoulin, thanks for the PR, just upgraded your rights to triage so you should be able to select reviewers.

jeremydumoulin commented:
Thank you all for processing this PR and for your valuable comments.
